### PR TITLE
update info on initial user creation

### PR DESCRIPTION
**Pull request by mchalski**
we're migrating from the 'create user' screen to useradm's CLI.

I'm also removing the troubleshooting section regarding password reset.
this used to be a real issue - in a single-user system, the user could
easily lock himself out of the system in case of forgotten credentials.

now a new user can simply be created, and soon user mgmt via the ui will
be added. the 'reset-user' script is slightly confusing ATM (will remove *all* users),
and can be removed from integration.

Issues: MEN-1037

Changelog: None

Signed-off-by: mchalczynski <marcin.chalczynski@rndity.com>

@maciejmrowiec @bboozzoo

### CODE REVIEW

**Comment by drewmoseley** [comment]
Acked-by: Drew Moseley \<drew.moseley@mender.io\>

@estenberg it looks like the code for this is already merged into the master branch: https://github.com/mendersoftware/useradm/pull/51

Any reason not to merge this now?

**Comment by drewmoseley** [comment]
@mchalski does this need to go into branch 1.1.x as well?

**Comment by mchalski** [comment]
@drewmoseley no, everything related to this is outside of 1.1

**Comment by estenberg** [comment]
@drewmoseley I am late but this looks good to me.

File: 201.Troubleshooting/04.Mender-Server/docs.md [hunk 1]
```diff
@@ -8,21 +8,6 @@ This document details troubleshooting steps for the most common problems with th
 The first part applies to all installations, while the section below on Production installations
 only applies when the Mender server is [installed for production](../../administration/production-installation).
 
-## Resetting administrative credentials
```
Comment (maciejmrowiec) [inline 201.Troubleshooting/04.Mender-Server/docs.md]: If this is going away, how do you rest password now ?
That script is still available in master.

**Pull request merged**